src/test/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/SplitNCigarReadsUnitTest.java

```diff
@@ -32,12 +38,58 @@
     };
 
     private final class TestManager extends OverhangFixingManager {
-        public TestManager( final SAMFileHeader header ) {
-            super(header, null, hg19GenomeLocParser, hg19ReferenceReader, 10000, 1, 40, false);
+        public TestManager( final SAMFileHeader header , DummyTestWriter writer) {
+            super(header, writer, hg19GenomeLocParser, hg19ReferenceReader, 10000, 1, 40, false, true);
         }
     }
 
     @Test
+    public void testBogusNSplits() {
+        DummyTestWriter writer = new DummyTestWriter();
+        final SAMFileHeader header = new SAMFileHeader();
```

Comment:
You could make this header a constant and then add a method to get an active manager.  That Would let you split up the tests without spending a lot of extra lines on boilerplate.